examples/wasi.rs

```diff
@@ -15,9 +15,11 @@
 //!
 //! Ready?
```

Comment:
Duplicate this test and do it the old way

Reply:
I added an example with the manual setup flow.